assets/scss/_cart.scss

```diff
@@ -31,7 +31,7 @@
   .giftsum-gift-row {
     @include respond-to(extrasmall) {
       display: block;
-      border-top: 2px solid #ddd;
+      // border-top: 2px solid #ddd;
```

Comment:
Why is a commented line being submitted with no information surrounding why it's commented. Either delete or provide information inline.